build/root/WORKSPACE

```diff
@@ -66,3 +50,10 @@ docker_pull(
     registry = "gcr.io",
     repository = "google-containers/debian-iptables-amd64",
 )
+
+docker_pull(
+    name = "official_busybox",
+    registry = "index.docker.io",
+    repository = "library/busybox",
+    tag = "latest",  # TODO: more hermetic?
```

Comment:
What if we fix the minor version instead? I.e. `1.26`? I'm worried that the hard-coding the sha is just going to create churn for us (and as this is only used as a base image, it won't have the same problems a moving tag does)